mkosi/tree.py

```diff
@@ -146,8 +148,10 @@ def copy_tree(
 
 def rmtree(*paths: Path, sandbox: SandboxProtocol = nosandbox) -> None:
     if paths:
+        parents = set(p.parent for p in paths)
```

Comment:
Let's make this
```suggestion
        parents = sorted(set(p.parent for p in paths))
```
since `set` is not guaranteed to be. This way we guarantee the output to look stable between different invocations.